src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts

```diff
@@ -166,6 +167,37 @@ export class AcrossConfigStoreClient extends BaseAbstractClient {
     return liteChainIdList.find((update) => update.blockNumber <= blockNumber)?.value ?? [];
   }
 
+  /**
+   * Resolves the lite chain ids that were available to the protocol at a given timestamp.
+   * @param timestamp Timestamp to search for. Defaults to latest time - in seconds.
+   * @returns List of lite chain IDs that were available to the protocol at the given timestamp.
+   * @note This dynamic functionality has been added after the launch of Across.
+   */
+  getliteChainIdIndicesForTimestamp(timestamp: number = Number.MAX_SAFE_INTEGER): number[] {
+    const liteChainIdList = sortEventsDescending(this.liteChainIndicesUpdates);
+    return liteChainIdList.find((update) => update.timestamp <= timestamp)?.value ?? [];
+  }
+
+  /**
+   * Checks if a chain ID was a lite chain at a given block.
+   * @param chainId The chain ID to check.
+   * @param blockNumber The block number to check. Defaults to latest block.
+   * @returns True if the chain ID was a lite chain at the given block. False otherwise.
+   */
+  isChainLiteChainAtBlock(chainId: number, blockNumber: number = Number.MAX_SAFE_INTEGER): boolean {
```

Comment:
Where is this function used?

Reply:
No longer used in the new update: removed here https://github.com/across-protocol/sdk/pull/663/commits/d5c74a91cd9cd7082ca5b4dce0931ecc6d39ad97